project.sh

```diff
@@ -433,19 +433,22 @@ require-no-focus(){
 }
 
 lein-lint () {
+	local alias
+	alias="$1"
+	require-var alias
 	if is-lein;then
 		require-no-focus
 		copy-to-project '.clj-kondo/config.edn'
-		echo-message 'Linting Clojure'
-		lein-dev lint
+		echo-message "Linting Clojure with alias '$alias'"
+		lein-dev "$alias"
```

Comment:
It's not really an alias. I'd prefer a name like `cmd`:
```suggestion
	local cmd
	cmd="$1"
	require-var cmd
	if is-lein;then
		require-no-focus
		copy-to-project '.clj-kondo/config.edn'
		echo-message "Linting Clojure with '$cmd'"
		lein-dev "$alias"
```
I'm not sure this suggestion will work since "Applying suggestions on deleted lines is not supported"

Reply:
Leiningen calls them aliases, so they are aliases `which are interpreted in the same way as command-line arguments to the lein command`. But they are also commands.

Renamed the var to `cmd`